packages/gated-content/src/conditions/index.ts

```diff
@@ -194,6 +197,19 @@ function toRawSimpleCondition(gqlCondition: gql.ThirdTierCondition): raw.SimpleC
         follow: raw.toProfileId(gqlCondition.follow),
       });
 
+    case 'AdvancedContractCondition':
+      return {
+        type: raw.ConditionType.ADVANCED_CONTRACT,
+        contract: toRawNetworkAddress(gqlCondition.contract),
+        abi: gqlCondition.abi,
+        functionName: gqlCondition.functionName,
+        params: gqlCondition.params as string[],
```

Comment:
is this because of a readonly?

Reply:
yeah, open to suggestions to clean this up